assets/src/components/activities/adaptive/AdaptiveAuthoring.scss

```diff
@@ -26,3 +26,215 @@
     padding-top: 64px !important;
   }
 }
+
+@import url('https://etx-nec.s3.us-west-2.amazonaws.com/css/torus/etx/fonts/Font+Awesome/FontAwesome6/css/all.css');
```

Comment:
shouldn't we use an official CDN? or check this into the repo separately? pointing to a ETX url in the open source project seems maybe problematic

Reply:
Opps, I just realized we don't even need this.